web/client-api/src/main/resources/io/deephaven/web/super/io/deephaven/util/MultiException.java

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.util;
+
+import java.util.List;
+
+/**
+ * Simplified version of MultiException, which doesn't require PrintWriter.
+ */
+public class MultiException extends Exception {
+
+    public static final Throwable[] ZERO_LENGTH_THROWABLE_ARRAY = new Throwable[0];
+    private final Throwable[] causes;
+
+    /**
+     * Create a MultiException from an array of Throwable causes.
+     *
+     * @param description the message to use
+     * @param causes a list of causes
+     */
+    public MultiException(String description, Throwable... causes) {
+        super(description, getFirstCause(causes));
+        this.causes = causes == null ? ZERO_LENGTH_THROWABLE_ARRAY : causes;
+    }
+
+
+    /**
+     * If there is a single exception, return that exception; otherwise wrap the causes into a MultiException.
+     *
+     * @param description the description for the MultiException
+     * @param causes the list of causes
+     * @return a MultiException or the single Throwable
+     */
+    public static Throwable maybeWrapInMultiException(String description, List<? extends Throwable> causes) {
+        if (causes.size() == 1) {
+            return causes.get(0);
+        }
+        return new MultiException(description, causes.toArray(ZERO_LENGTH_THROWABLE_ARRAY));
+    }
+
+    private static Throwable getFirstCause(Throwable[] causes) {
+        if (causes == null || causes.length == 0) {
+            return null;
+        }
+
+        return causes[0];
+    }
+
+    /**
+     * @return all of the exceptions that resulted in this one.
+     */
+    public Throwable[] getCauses() {
+        return causes;
+    }
+
+    @Override
+    public String getMessage() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(super.getMessage()).append(": \n");
```

Comment:
Don't think space should be here
```suggestion
        sb.append(super.getMessage()).append(":\n");
```

Reply:
Possibly, but I'm keeping as close to the original implementation as possible, only removing some blocking IO that the client can't support:
https://github.com/deephaven/deephaven-core/blob/b5cae98c2f11b032cdd1b9c248dc5b4a0f95314a/Util/src/main/java/io/deephaven/util/MultiException.java#L108